README.md

```diff
@@ -31,3 +31,45 @@ const experimentPages = [
     }
 ];
 ```
+
```

Comment:
Maybe add a title here like `### Defining a route for the worker`?

Reply:
Would linking to https://developers.cloudflare.com/workers/about/routes/#matching-behavior be good here as a primer for configuring routes in general?